service/gcs/stdio/stdio.go

```diff
@@ -291,12 +291,19 @@ func (pr *PipeRelay) closePipes() {
 }
 
 // NewTtyRelay returns a new TTY relay for a given master PTY file.
-func NewTtyRelay(s *ConnectionSet, pty *os.File) *TtyRelay {
-	return &TtyRelay{s: s, pty: pty}
+func NewTtyRelay(s *ConnectionSet, pty *os.File, isInit bool) *TtyRelay {
```

Comment:
I think you're pushing too much business logic down here. The TTY relay shouldn't have to care about the initial process vs subprocesses.

Should this be named something like `ignoreClosedError` or something?

Reply:
Alternatively, should we just always ignore errors?

Reply:
The containerd console has an explicit call to close on the fd when they issue the delete. Our relay automatically closes the fd after a call to wait. So we either need to handle init differently or just ignore the errors in general. I am fine with just ignoring the error on closed in general if you want. But its "most correct" to only ignore it for the init process. But since GCS has different lifetimes it doesnt really map

Reply:
I'm open to either, but this relay's interface should not be tied to the concept of initial vs non initial processes, right? If we wanted to reuse this relay elsewhere, `isInit` would be meaningless.

Reply:
We will just ignore them for all closed tty's then

Reply:
OK that's fine, but I'm not sure if I'm being clear--you can also just rename `isInit` to something that means what it does instead of what scenario it's used in.